kolibri/core/assets/src/state/actions.js

```diff
@@ -223,6 +223,36 @@ function getCurrentSession(store) {
   .catch(error => { handleApiError(store, error); });
 }
 
+function getFacilityConfig(store) {
+  const coreApp = require('kolibri');
+  const FacilityCollection = coreApp.resources.FacilityResource
+    .getCollection()
+    .fetch();
+
+  return FacilityCollection.then(facilities => {
+    const currentFacilityId = facilities[0].id;
+    const facilityConfigCollection = coreApp.resources.FacilityDatasetResource
+      .getCollection({ facility_id: currentFacilityId })
+      .fetch();
+    return facilityConfigCollection
```

Comment:
:+1: I think that's a good idea, since it codifies what most of these API -> vuex mappers do anyway (renaming the keys). The one downside, though, is that it hides useful type information about what is being returned from the backend and sent to the view. 

We should try using it in more places to see how it goes, but I can see this removing a lot of boilerplate code if it works out.